beacon-chain/sync/initial-sync/blocks_fetcher.go

```diff
@@ -0,0 +1,336 @@
+package initialsync
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/kevinms/leakybucket-go"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/pkg/errors"
+	eth "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/beacon-chain/flags"
+	"github.com/prysmaticlabs/prysm/beacon-chain/p2p"
+	prysmsync "github.com/prysmaticlabs/prysm/beacon-chain/sync"
+	p2ppb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/mathutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/sirupsen/logrus"
+)
+
+// blocksFetcherConfig is a config to setup block fetcher
```

Comment:
```suggestion
// blocksFetcherConfig is a config to setup the block fetcher.
```
Make sure comments are always full sentences with periods 😄